eradiate/validators.py

```diff
@@ -47,26 +49,39 @@ def path_exists(_, attribute, value):
     an existing target. Raises a ``FileNotFoundError`` otherwise.
     """
     if not value.exists():
-        raise FileNotFoundError(f"{attribute} points to '{str(value)}' "
-                                f"(path does not exist)")
+        raise FileNotFoundError(
+            f"{attribute} points to '{str(value)}' " f"(path does not exist)"
```

Comment:
```suggestion
            f"{attribute} points to '{str(value)}' (path does not exist)"
```

Reply:
I didn't actually reformat these manually, I think black removed the line breaks, that's why the format is so strange.